tikv/raftstore/db_writer.go

```diff
@@ -14,20 +21,115 @@ func (writer *raftDBWriter) Close() {
 }
 
 type raftWriteBatch struct {
+	requests []*rfpb.Request
 	startTS  uint64
 	commitTS uint64
 }
 
+// transferLockToValue decode the mvcc lock and returns putLock value and putDefault value if exists.
+func transferLockToValue(lock *mvcc.MvccLock) ([]byte, []byte, error) {
```

Comment:
`EncodeLockCFValue` is better.
And this can be moved to `mvcc` package, similar to `EncodeWriteCFValue`.